code/modules/mob/living/say.dm

```diff
@@ -101,7 +101,14 @@ proc/get_radio_key_from_channel(var/channel)
 		verb = pick("slobbers","slurs")
 		speech_problem_flag = 1
 	if(stuttering)
-		message = stutter(message)
+		if(iscarbon(src))

```

Comment:
This is OOP-murder. You should never run `istype(src, ...))`. Should extract this into a function maybe.

```dm
/mob/living/proc/get_stuttered_message(message)
	return stutter(message)

/mob/living/carbon/get_stuttered_message(message)
	if (shock_stage >= 30)
		return stutter(message)
	else
		return NewStutter(message)
```